package.json

```diff
@@ -1,54 +1,85 @@
 {
   "name": "active-model-adapter",
   "version": "2.2.0",
-  "repository": "https://github.com/adopted-ember-addons/active-model-adapter",
-  "description": "Adapters and Serializers for Rails's ActiveModel::Serializers",
+  "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "",
+  "license": "MIT",
+  "author": "",
```

Comment:
we should probably replace this with `contributors`

Reply:
Ought this field to contain you, Igor and myself?

Reply:
I'm not sure if adopted-ember-addons has a convention, but the top 8 here plus Igor/Myself and you would cover everyone: https://github.com/adopted-ember-addons/active-model-adapter/graphs/contributors?from=2015-06-07&to=2021-04-19&type=c

You might have to search the git history for their emails. I'm fine with this being done in follow up.